client/client_test.go

```diff
@@ -84,6 +87,27 @@ func (s *ClientSuite) SetUpTest(c *C) {
 	for k, v := range targetFiles {
 		s.remote["targets/"+k] = newFakeFile(v)
 	}
+
+	s.expiredTime = time.Now().Add(time.Hour)
+}
+
+// withMetaExpired sets signed.IsExpired throughout the invocation of f so that
+// any metadata marked to expire at s.expiredTime will be expired (this avoids
+// the need to sleep in the tests).
+func (s *ClientSuite) withMetaExpired(f func()) {
+	signed.IsExpired = func(t time.Time) bool {
+		return t.Unix() == s.expiredTime.Unix()
+	}
+	f()
+	signed.IsExpired = func(time.Time) bool { return false }
```

Comment:
For consistency, this should also be set in an `init()` function for the test suite, otherwise tests that run after a call to this could behave differently.